docs/r/versions.json

```diff
@@ -4,7 +4,7 @@
         "version": "dev/"
     },
     {
-        "name": "11.0.0 (release)",
+        "name": "11.0.0.3 (release)",
```

Comment:
Do we need this change in apache/arrow too?
Can we automate this in our release process?

Reply:
This is due to an R release separate from the main release, and some version number incrementing that might have not been necessary in the end.  I'll add it to the R release checklist.